frontend-react/src/utils/OrganizationUtils.test.ts

```diff
@@ -1,4 +1,43 @@
-import { groupToOrg } from "./OrganizationUtils";
+import { AccessToken } from "@okta/okta-auth-js";
+
+import {
+    getOktaGroups,
+    getRSOrgs,
+    groupToOrg,
+    parseOrgs,
+    RSOrgType,
+    RSUserClaims,
+} from "./OrganizationUtils";
+
+const badAccessToken: AccessToken = {
+    authorizeUrl: "",
+    expiresAt: 0,
+    scopes: [],
+    userinfoUrl: "",
+    accessToken: "",
+    claims: {
+        sub: "", // This satisfies the required attribute
+    },
+    tokenType: "",
+};
+
+const goodAccessToken: AccessToken = {
```

Comment:
If I've got the time, I would love to try making a fake AccessToken generator, so the only values we need to pass in are the values we want to mock. Let the defaults populate themselves and keep things to a one-liner:

```typescript
const goodAccessToken: AccessToken = mockToken({ organization: ["DHSender_xx-phd", "DHxx_phd"] })
```